homeassistant/components/fivem/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "FiveM server is already configured"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect. Please check the host and port and try again. Also ensure that you are running the latest FiveM server."
```

Comment:
Please dont make any edits to `translations/en.json`, just edit the strings.json and generate translations via `python3 -m script.translations develop`.

Reply:
I did not know about this. Thanks for telling me I will use this in future commits

Reply:
If you use the vscode devcontainer/setup, a compile translations task has been added to make it easier to do this.